README.md

```diff
@@ -1,17 +1,16 @@
 # PSPDFKit for iOS Appcelerator Bindings
 
-PSPDFKit - The Leading Mobile PDF Framework for iOS and Android.
+PSPDFKit - The Leading Mobile PDF Framework for iOS and Android.  PSPDFKit 9 for iOS needs at least Xcode 11 or higher and supports iOS 11 ([read more](https://pspdfkit.com/guides/ios/current/announcements/version-support/)). 
 
 ## How to build
 
 **Note:** Read the [Alloy](#alloy) section before building the module if you're using the Alloy framework.
 
 1. Checkout the project: `git clone https://github.com/PSPDFKit/Appcelerator-iOS.git`
-2. Make sure you have Titanium 8.0.1.GA or later installed: https://www.appcelerator.com or `ti sdk install 8.0.1.GA`
+2. We recommend using Titanium SDK version 8.2.1GA, or later. Follow the installation instructions over at [https://github.com/appcelerator/titanium](https://github.com/appcelerator/titanium).
 3. Download the binary build of PSPDFKit from [your customer portal](https://customers.pspdfkit.com)
-4. Open the downloaded .dmg and copy `PSPDFKit.framework` and `PSPDFKitUI.framework` into the `platform/` folder inside the project.
-4a. Optionally, copy the `PSPDFKit.bundle` from `PSPDFKit.framework` to `Resources/`
-5. Call `ti build -p ios --build-only` in the `Appcelerator-iOS` folder.
+4. Inside the `.dmg`, you'll find `PSPDFKit.**xcframework**` and `PSPDFKitUI.**xcframework**` (read more about our use of the new `xcframework` format [here](https://pspdfkit.com/blog/2019/pspdfkit-ios-9/)). Copy the contents of the `ios-arm64` folder of both `xcframeworks` folders into the `platform/` folder within this repository. Optionally, also copy `PSPDFKit.bundle` inside `PSPDFKit.framework` to `Resources/`.
```

Comment:
I assume that you are using the the `.frameworks` and not the `xcframeworks` because Titanium forces us to use the Legacy Build System which does not support XCFramework.

If that's the case, the I would suggest to use the fat frameworks from the CocoaPods artifacts, this way it will work on both real iOS devices and the simulator.

The link to download the `.frameworks` directly can be found in the podspec (https://customers.pspdfkit.com/cocoapods/YOUR_COCOAPODS_KEY/pspdfkit/latest.podspec). I would use this link here in the integration instructions: https://customers.pspdfkit.com/cocoapods/YOUR_COCOAPODS_KEY/pspdfkit/latest.zip

Reply:
Wait, Titanium should be fine with the new build system. Is there a known issue with it?

Reply:
Just tested with the new build system (the default now) and everything seems to be working just fine. 👍